seg_utils/src/label_main.py

```diff
@@ -47,9 +55,10 @@ def __init__(self):
 
         # color stuff
         self._num_colors = 25  # number of colors
-        self.colorMap = None
+        self.colorMap = None  # type: list
```

Comment:
at the top of the file add the import `from typing import List`
then change this to `# type: List[QColor]`
You should do this for any lists that you have type hints for. This can also be done for dict and other things. This is nice because when you get an item from the list, you (pycharm) will know what it is.